.github/workflows/ansible_lint.yml

```
@@ -14,22 +13,10 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: install dependencies
-        run: pip3 install ansible ansible-lint yamllint molecule
-
-      - name: install roles from galaxy
-        run: ansible-galaxy install elan.opencast_repository
-
-      - name: create lxc container
-        uses: lkiesow/setup-lxc-container@v1
-        with:
-          dist: centos
-          release: 9-Stream
+        run: pip3 install ansible ansible-lint yamllint
```

We don't actually need ansible any longer, do we?
```suggestion
        run: pip3 install ansible-lint yamllint
```

---

Yes, ansible-lint should install dependencies on ansible-core by itself anyway. Changed.